### PR TITLE
sample workflow for namespaced scope installation.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

## Proposed changes

Adding sample workflow for namespaced scope installation.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- None.

## Special notes for your reviewer:
Workflow for Okteto and similar namespaced scope installations of litmus-portal.

### CODE REVIEW

**Review by rberrelleza — APPROVED** [review]

**Comment by ishangupta-ds** [comment]
@oumkale @S-ayanide @arkajyotiMukherjee @Jonsy13 @amityt @Saranya-jena @rajdas98 PTAL.

**Review by ksatchit — APPROVED** [review]

File: litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts [hunk 1]
```diff
@@ -116,13 +113,9 @@ export default [
       'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/pod-delete/workflow.yaml',
     chaosWkfCRDLink_Recur:
       'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/pod-delete/workflow_cron.yaml',
-
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts:119]: Why is the git link and isCustom parameters removed? @ishangupta-ds

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts:119]: Added them back @Jonsy13

**Comment by ishangupta-ds** [comment]
PTAL @Jonsy13 @rajdas98 @gdsoumya @arkajyotiMukherjee @oumkale

**Review by Jonsy13 — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**